### PR TITLE
Add noop package for Python 3.8+

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

As suggested in https://github.com/conda-forge/importlib_metadata-feedstock/pull/47#issuecomment-705674328

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,8 +11,14 @@ source:
   sha256: 77a540690e24b0305878c37ffd421785a6f7e53c8b5720d211b211de8d0e95da
 
 build:
-  number: 1
   noarch: python
+{%- if build_script == 'noop' %}
+  number: {{ build_number | int * 2 + 1 }}
+  script: {{ PYTHON }} --version
```
Comment (mariusvniekerk) [inline recipe/meta.yaml:17]: since your base doesn't depend on ptyhon this won't work

**Comment by chrisburr** [comment]
For some reason it isn't rendering with the `build_script` variants...

**Comment by chrisburr** [comment]
Ah ha! Conda smithy doesn't like my use of `{%- ` instead of `{% `.

**Comment by chrisburr** [comment]
Ah I hadn't realised `importlib_metadata` doesn't provide an identical name to the Python 3.8 standard library. I think this needs to be left as it currently stands in master rather than claiming `importlib_metadata` is installed while `importlib_metadata` isn't importable.

What do you think @mariusvniekerk? If you agree, can you push a commit to master as the CI didn't trigger for #47's merge commit.

**Comment by mariusvniekerk** [comment]
ugh well thats annoying

**Pull request closed without merging**